applications-code-snippets/python-hashkey-generator/readme.md

```diff
@@ -0,0 +1,21 @@
+## About Admission System Models
+
+Todo from @[Mubtasim Fuad](https://github.com/mubtasimfuad) if he's available 😛
```

Comment:
Hello @mubtasimfuad Bhai👋

Interested in Hacktoberfest this year? You can start by making a PR to update this README. That'll be your first PR, then you can do 3 more as per your wish.

Thanks!